core/http/Cargo.toml

```diff
@@ -58,3 +58,4 @@ features = ["std"]
 
 [dev-dependencies]
 rocket = { version = "0.5.0-dev", path = "../lib" }
+serde_json = "*"
```

Comment:
Never use `*` dependencies. We _must_ use a specific version. Otherwise, our crate can be broken by upstream changes.

Reply:
I have changed it to 1.0, to match serde